apps/auth/utils.py

```
@@ -0,0 +1,40 @@
+import bcrypt
+from typing import Dict, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)